sbnana/SBNAna/Cuts/NueCuts.cxx

```diff
@@ -134,8 +134,9 @@ namespace ana{
   const Cut kNueAllTrackDazzleMuonCut(
     [](const caf::SRSliceProxy* slc)
     {
-      for (auto const& trk : slc->reco.trk)
+      for (auto const& pfp : slc->reco.pfp)
       {
+      const auto& trk = pfp.trk;
```

Comment:
I think this is missing a check on the track score if we want to be completely backwards compatible